flathub-external-data-checker/entrypoint.sh

```diff
@@ -7,6 +7,14 @@ fi
 
 detect_manifest() {
     repo=${1}
+    
+    # check if repo opted out
+    if [[ -f $repo/flathub.json ]]; then
+        jq -e '."disable-external-data-checker"' < $repo/flathub.json
+        if [ $? -ne 0 ]; then
```

Comment:
It's the other way round, I guess? `jq` exiting with non-zero status means `disable-external-data-checker` is _not_ `true`.

Reply:
🤦 of course